testmodule/providers/jsonrpc/testConversions.nim

```diff
@@ -17,14 +20,16 @@ suite "JSON Conversions":
       "timestamp":"0x6285c293"
     }
 
-    var blk = Block.fromJson(json)
-    check blk.number.isNone
+    without blk1 =? Block.fromJson(json):
+      fail
```

Comment:
I'd do this:

```suggestion
    without blk1 =? Block.fromJson(json):
      fail()
```
Or even this:
```suggestion
    var blk = !Block.fromJson(json)
```

I don't like using `!`, except in tests like this.